src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml

```diff
@@ -89,7 +88,11 @@ the MIT License. See LICENSE in the project root for license information. -->
                             Style="{StaticResource BrowseButtonStyle}"
                             Click="Rename_Click"
                             IsEnabled="{x:Bind CanDeleteCurrentScheme, Mode=OneWay}">
-                        <FontIcon Glyph="&#xE8AC;"/>
+                        <StackPanel Orientation="Horizontal">
+                            <FontIcon Glyph="&#xE8AC;"
+                                      FontSize="{StaticResource StandardIconSize}"/>
+                            <TextBlock/>
+                        </StackPanel>

```

Comment:
...

You know this ain't gonna fly... Boot up XAML Studio and find a padding/margin to make this look right and remove this hack.

We've gotten a number of complaints (from Niksa, mostly) about the buttons' top/bottom not being aligned with the top/bottom of the ComboBox/TextBox they're next to. This would be a good opportunity to fix that once and for all.